File: src/test/java/org/skefir/data/EventFilteredCondition.java
package org.skefir.data;

import lombok.Value;

import java.util.Set;

/**
 * класс данных набора фильтров календаря
 */
@Value
public class EventFilteredCondition {
    Set<ImportanceFilterOption> importanceSet;
    DateFilterOptions dateFilterOption;
    Set<Currencies> currenciesSet;
}

File: src/test/kotlin/org/skefir/entity/EventFilteredCondition.kt
package org.skefir.entity

import org.skefir.data.Currencies
import org.skefir.data.DateFilterOptions
import org.skefir.data.ImportanceFilterOption

data class EventFilteredCondition(
    var importanceSet: Set<ImportanceFilterOption>,
    var dateFilterOption: DateFilterOptions,
    var currenciesSet: Set<Currencies>)
